Arena.toml

```diff
@@ -1170,11 +1235,26 @@ document = "stjudecloud/workflows:/tools/util.wdl"
 message = "util.wdl:713:1: note[LineWidth]: line exceeds maximum width of 90"
 permalink = "https://github.com/stjudecloud/workflows/blob/129060215443bbd1b67c4d3d149d48acdab90ef2/tools/util.wdl/#L713"
 
+[[diagnostics]]
+document = "stjudecloud/workflows:/workflows/chipseq/chipseq-standard.wdl"
+message = "chipseq-standard.wdl:102:9: note[BlanksBetweenElements]: missing blank line"
+permalink = "https://github.com/stjudecloud/workflows/blob/129060215443bbd1b67c4d3d149d48acdab90ef2/workflows/chipseq/chipseq-standard.wdl/#L102"
```

Comment:
What's causing this? I don't see anywhere that needs a blank.

Reply:
There is no whitespace between the private declaration and the preceding call. The description in #11 is unclear.

> Note the blank lines between meta, parameter meta, input, the first call or first private declaration, output, and runtime for the example task. 

I interpret that as each block of `call`s or private declarations needs a preceding blank line.

Reply:
It also says 

> There should be a blank line between each WDL element at the root indentation level (such as the import block and any task/workflow definitions) and between sections of a WDL task or workflow.

I interpret `sections` to include changing between `scatter`, `call`, and private declaration blocks.

Reply:
O ok. I agree #11 is vaguely worded.

I'll detail what I meant in #11, although I'm open to enforcing something different:

What I meant was that "the body" (beginning with the first declaration, call statement, conditional,  or scatter) should have a blank line between it and the prior section (which should be the `input` section). I didn't intend to draw any distinction between `scatter` statements, `call` statements, `if` statements, and declarations. In my mind they all equally contribute to a "workflow body". And in addition, a workflow body is allowed to include singular blank lines between any of the statements at the author's discretion. Similarly writing a workflow body without _any_ blank lines was meant to be permitted.

So my intention was much more permissive than your interpretation.

Reply:
@a-frantz - That would've been a much easier implementation as well.

Reply:
😅 
I think _there is_ a case to be made that what I wrote was too permissive, and we should enforce something closer to what you implemented. To take each example to the extreme, if a complicated workflow like the `quality_check` "body" didn't have any blank lines, it would be really hard to read. It would be a similar issue if every clause was whitespace separated. So there's a happy medium out there, I just don't know the right logic for arriving at it. Which is why I passed on enforcing anything specific and left it up to authors.

Reply:
I just pushed the update. Now I only check to see if the `input` block is the prior element to a `if`, `scatter`, or non-input, non-output declaration. If so, it requires a blank line. Otherwise, I do nothing.

Reply:
Once (1) is finished. I don't think that (2) matters. If any non-body element is found, then there needs to be a preceding blank line.